widget/table.go

```diff
@@ -67,6 +91,50 @@ func (t *Table) CreateRenderer() fyne.WidgetRenderer {
 	return r
 }
 
+// SetSelected will mark the specified cell (at row, col) to be marked as selected.
+func (t *Table) SetSelected(row, col int) {
```

Comment:
nit: #1395
```suggestion
func (t *Table) SetSelectedCell(row, col int) {
```

Reply:
So under this suggestion then list will need to be changed from SetSelected to SetSelectedItem?

Reply:
This was following the already-in API was it not? I didn't think I was making any choices here.

Reply:
List had SelectedItem field and Tree has SetSelectedNode

Reply:
> This was following the already-in API was it not?

This API hasn't hit master yet, so now is the time to get consistency

Reply:
Indeed, I was just referencing what you guys already had pushed to develop.
I didn't realise that we were adapting these PRs to #1395, I assumed that because it needed more discussion we would be addressing it later. It covers many widgets afterall.

Reply:
I think this now matches the updated discussion